src/components/Reusables/PageContainer/PageContainer.module.css

```diff
@@ -0,0 +1,12 @@
+.pageWrapper{
+  width: 1055px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 1rem;;
+  background-color: #1EADF7;
```

Comment:
Should this be a token from the design system or something?

Reply:
The figma tokens are only supposed to be consumed by the design system for now. Maybe in future we can base our color on the figma tokens when they are ready to be consumed by other altinn apps.